momepy/functional/_distribution.py

```diff
@@ -339,3 +344,124 @@ def street_alignment(
     Series
     """
     return (building_orientation - street_orientation.loc[street_index].values).abs()
+
+
+def cell_alignment(
+    left_orientations: np.ndarray | Series, right_orientations: np.ndarray | Series
+):
+    """
+    Calculate the difference between cell orientation and the orientation of object.
+
+    .. math::
+        \\left|{\\textit{building orientation} - \\textit{cell orientation}}\\right|
+
+    Parameters
+    ----------
+    left_orientations : np.array, pd.Series
+        The  ``np.array``, or
+        `pd.Series`` where object orientation values are stored. This
+        can be calculated using :func:`orientation`.
+    right_orientations : np.array, pd.Series
+        The ``np.array`` or ``pd.Series``
+        where object orientation values are stored. This
+        can be calculated using :func:`orientation`.
+
+    Returns
+    -------
+    Series
+    """
+    return (left_orientations - right_orientations).abs()
+
+
+def shared_walls_ratio(
+    shared_walls: np.ndarray | Series, perimeters: np.ndarray | Series
+):
+    """Calculate shared walls ratio of adjacent elements (typically buildings).
+
+    .. math::
+        \\textit{length of shared walls} \\over perimeter
+
+    Note that data needs to be topologically correct.
+    Overlapping polygons will lead to incorrect results.
+
+    Adapted from :cite:`hamaina2012a`.
+
+    Parameters
+    ----------
+    shared_walls : np.array, pd.Series
+        The  ``np.array``, or
+        `pd.Series`` shared wall polygons are stored. This
+        can be calculated using :func:`shared_walls`.
+    perimeters : np.array, pd.Series
+        The  ``np.array``, or
+        `pd.Series`` containing used perimeters values.
+        Typically the perimeters of the original GeoDataFrame
+        used to generate the shared walls.
+
+    Returns
+    -------
+    Series
+    """
+
+    return shared_walls / perimeters
```

Comment:
I'd remove this and point users to `shared_walls / perimeters` directly. It is a fairly obvious one-liner